### DISCUSSION TITLE
Fps_Out 不理解此参数

### DISCUSSION THREAD

**Discussion by jian-jiu** [hooke007/mpv_PlayKit#514] [Q&A 问答]
配置文件写的是
## 浮点，目标帧率，示例即约60fps。特殊值比如 display_fps/2 即目标为显示器刷新率的一半帧率；填 container_fps*2 即倍帧（越高的帧率增加耗能，产生更多伪影，通常是不必要的。例如对于144hz的显示器来说，最多只需要补到72fps，剩下的可让 --interpolation 帧混成补偿）

如果是 144hz的显示器，配置 Fps_Out = 72.0 ， interpolation = yes（默认就是开启的）
我看统计信息显示的是 Frame Rate: 30fps（specified）72fps（estimated）
那我看见的实际视频帧率是144吗？

也就是说如果开启了 帧混成补偿 实际看见的帧率 是 统计信息里面的 estimated * 2 吗?

**Comment by hooke007** [comment]
https://github.com/hooke007/MPV_lazy/blob/d16afd57f72f57c9f9bbf504ab43f435ec06b804/portable_config/mpv.conf#L183
interpolation在我版本里自动化处理的，补帧的情况下自动关闭，见profiles.conf
https://github.com/hooke007/MPV_lazy/blob/b6b10a53fe86297ed0eef555172331b24bedcd08/portable_config/profiles.conf#L27-L28

estimated fps如果显示就是当前的预估输出帧率

**Reply by hooke007** [reply]
https://github.com/hooke007/MPV_lazy/wiki/3_K7sfunc#mvt_std
fps_out 就是此模块的输出帧率。

示例脚本里补充的建议，目的是节省性能。
本来interpolation应该开启，只不过现在有bug反倒恶化性能
https://github.com/hooke007/MPV_lazy/discussions/385
